js/admin.block-post.js

```diff
@@ -18,11 +18,16 @@
 		$( '#block-add-field' ).on( 'click', function() {
 			let template = wp.template( 'field-repeater' ),
 				data     = { uid: new Date().getTime() },
-				field    = $( template( data ) );
-			$( '.block-fields-rows' ).append( field );
+				field    = $( template( data ) ),
+				edit     = field.find( '.block-fields-actions-edit' ),
+				label    = field.find( '.block-fields-edit-label input' );
+
+				$( '.block-fields-rows' ).append( field );
```

Comment:
Maybe this should be 1 tab to the left, not being part of the `let` block above.

Reply:
Nice pickup. Thanks. Fixed in 57d161c.